.gitignore

```diff
@@ -76,3 +76,4 @@ _configtest.o.d
 sklearn/utils/seq_dataset.pyx
 sklearn/utils/seq_dataset.pxd
 sklearn/linear_model/sag_fast.pyx
+venv/
```

Comment:
Please revert this change

Reply:
Kindly elaborate @thomasjpfan?
Thank you.

Reply:
You should remove the line. Even if this is useful locally, we don't want this change for all sklearn developer because it might not apply to them. You just need to remove (revert) this line and commit and push and the diff will disappear from GitHub.

Reply:
Okay. Thank you. Let me.